docs/examples/camera_display.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class MyCameraDisplay(CameraDisplay):
</pre><p>Side note: we should really change this to something like <code>enable_pixel_picker(callback)</code> instead of using inheritance </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/ctapipe/pull/2266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
agreed. It probably also works now if you just re-assign the function, like
```py3
disp.on_pixel_clicked = some_new_func
```
But I don't really want to encourage that